packages/react-native-bpk-theming/src/BpkThemeAttributes.js

```diff
@@ -89,6 +89,11 @@ export type Props = {
  *
  */
 const BpkThemeAttributes = (props: Props) => {
+  // eslint-disable-next-line no-console
```

Comment:
I think we have a helper for this warning, but maybe that was only for deprecated proptypes

Reply:
Will have a look, for this I just took the same logic from the `util.js` file where `grayForTheme` was deprecated

Reply:
Only seems to exist for propTypes in `packages/react-native-bpk-component-text/src/common-types.js` so not even a global function